File: 0783/783.cs
public class Solution {
    public int MinimumDistanceBetweenBstNodes(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MinimumDistanceBetweenBstNodes(root.left), MinimumDistanceBetweenBstNodes(root.right));
    }
}

File: 0783/783.cpp
class Solution {
public:
    int minimumDistanceBetweenBstNodes(TreeNode* root) {
        if(!root) return 0;
        return 1+max(minimumDistanceBetweenBstNodes(root->left),minimumDistanceBetweenBstNodes(root->right));
    }
};